encore/events/event_manager.py

```diff
@@ -250,7 +258,7 @@ def get_listeners(self, event):
                 listener = linfo[-1]
                 id = self.get_id(listener())
                 if id in l_filter:
-                    for key, value in l_filter[id].iteritems():
+                    for key, value in six.iteritems(l_filter[id]):
```

Comment:
I'd be willing to live with the simpler `.items()` here (even though less efficient on 2.7) unless we _know_ that creating an intermediate list is a bottleneck for a real application.